src/memusage.h

```diff
@@ -84,8 +84,8 @@ struct stl_shared_counter
     size_t weak_count;
 };
 
-template<typename X>
-static inline size_t DynamicUsage(const std::vector<X>& v)
+template<typename X, typename Y>
+static inline size_t DynamicUsage(const std::vector<X, Y>& v)
 {
     return MallocUsage(v.capacity() * sizeof(X));
```

Comment:
Right, it can,  when `reserve()` is called with a larger amount, or when the vector is shortened. But it does represent the actually-allocated size which seems what we want here.